### DISCUSSION TITLE
Can't add users to group

### DISCUSSION THREAD

**Discussion by DeCodie** [kanboard/kanboard#5332] [Installation & Configuration]
### Actual behaviour

Upon trying to add members to a group, the users do not get added as expected. There is no visible field to select or add group members. 

### Expected behaviour

When attempting to add members to a group within the Kanban Board, the selected users should be added to the group successfully. Their profiles or avatars should be displayed as part of the group, confirming their inclusion.

### Screenshots

![Capture10](https://github.com/kanboard/kanboard/assets/136938274/6049540d-1191-40ca-aeb3-987c8afab0d9)

### Configuration

- Kanboard version: 1.2.32
- Database type and version: MariaDB
- PHP version: 8.1.18
- OS: Linux 
- Browser: Chrome Version 114.0.5735.199

**Comment by DeCodie** [comment]
Any help @fguillot, @Chaosmeister?

**Comment by Chaosmeister** [comment]
heyo  
do you have other groups where this works?

**Comment by DeCodie** [comment]
Hi @Chaosmeister, No others are also acting the same way!

**Comment by fguillot** [comment]
I cannot reproduce your issue with Kanboard 1.2.32. Your problem is probably related to an incompatible plugin or theme.

**Reply by DeCodie** [reply]
Hello @fguillot, That's correct, I tried to disable the theme **[Moon](https://github.com/kenlog/Moon)** and it worked. Many thanks for your help and support!
